modules/ROOT/pages/indexes/semantic-indexes/vector-indexes.adoc

```diff
@@ -424,6 +557,25 @@ Cosine similarity rebounds the trigonometric cosine.
 image::cosine_similarity_equation.svg["The cosine of vector v and vector u is defined as half of the quanity 1 plus the scalar product of v hat u hat, which equals half of the quantity 1 plus the scalar product of vector v vector u over the product of the l2-norm of vector v and the l2 norm ov vector u, which exists in the bounded set of real numbers between 0 inclusive and 1 inclusive."]
 In the above equation the trigonometric cosine is given by the scalar product of the two unit vectors.
 
+[[indexes-vector-provider-differences]]
+== Compatibility index provider differences
+The default and preferred vector index provider is `vector-2.0`.
```

Comment:
Can we mention something brief about the differences between the 2 providers? What has improved with 2.0?

```suggestion
As of Neo4j 5.18, the default and preferred vector index provider is `vector-2.0`.
```